ddsmt.py

```diff
@@ -215,10 +213,9 @@ def _process_substitutions(pool, exprs, superset, superset_substs):
                     restart = True
                     break
 
-                _msg(2,
```

Comment:
Is there a way to have the same behavior as before with logging? This updates the current line instead of printing a new one.

Reply:
Hm, not easily. Maybe we can just use `sys.stdout.write()` manually at this one point. It is kind of special anyway...

Reply:
Yes, sounds good, it's the only place we need it.